train.py

```diff
@@ -84,14 +84,14 @@ def train_net(net,
 1. When do BasicDataset function(in src.unet_.dataset.py), image shpae become 5 dimension (2, 19, 681, 1086, 12) (batch size, yr, y, x, band).
 But the dimension in the model is 4 dimension (64, 3, 3, 3);
```

Comment:
Does the Unet model you're using have an option to change the number of channels and intput dim?

Reply:
The Unet model can change the number of channels and the number of classes, but I don't think it can change the input dim